internal/tests/cli/boundary/target_host_sources.bats

```diff
@@ -0,0 +1,214 @@
+load _target_host_sources
+load _targets
+load _host_catalogs
+load _host_sets
+
+export NEW_HOST1='test-for-add-host-source-1'
+export NEW_HOST2='test-for-add-host-source-2'
+export NEW_HOST_CATALOG='test-host-catalog'
+export NEW_HOST_SET1='test-host-set-1'
+export NEW_HOST_SET2='test-host-set-2'
+export TGT_NAME='test-target'
+export TGT_DEFAULT_PORT='22'
+
+@test "boundary/login: can login as admin user" {
+    run login $DEFAULT_LOGIN
+    [ "$status" -eq 0 ]
+}
+
+@test "boundary/target: admin user can create target" {
+    run create_tcp_target $DEFAULT_P_ID $TGT_DEFAULT_PORT $TGT_NAME
+    echo $output
+    [ "$status" -eq 0 ]
+}
+
+@test "boundary/host-catalogs: can create $NEW_HOST_CATALOG host catalog in default project scope" {
+    run create_host_catalog $NEW_HOST_CATALOG $DEFAULT_P_ID
+    echo "$output"
+    [ "$status" -eq 0 ]
+}
+
+@test "boundary/hosts: can multiple hosts in created host catalog" {
+    local hcid=$(host_catalog_id $NEW_HOST_CATALOG $DEFAULT_P_ID)
+    run create_host $NEW_HOST1 $hcid
+    echo "$output"
+    [ "$status" -eq 0 ]
+
+    run create_host $NEW_HOST2 $hcid
+    echo "$output"
+    [ "$status" -eq 0 ]
+}
+
+@test "boundary/host-sets: can add multiple hosts to a created host catalog" {
+    local hcid=$(host_catalog_id $NEW_HOST_CATALOG $DEFAULT_P_ID)
+    run create_host_set $hcid $NEW_HOST_SET1
+    echo "$output"
+    [ "$status" -eq 0 ]
+
+    run create_host_set $hcid $NEW_HOST_SET2
+    echo "$output"
+    [ "$status" -eq 0 ]
+}
+
+@test "boundary/host-set/add-host: can associate multiple host sets with created hosts" {
+    local hcid=$(host_catalog_id $NEW_HOST_CATALOG $DEFAULT_P_ID)
+    local hid1=$(host_id $NEW_HOST1 $hcid)
+    local hid2=$(host_id $NEW_HOST2 $hcid)
+
+    local hsid1=$(host_set_id $NEW_HOST_SET1 $hcid)
+    run assoc_host_set_host $hid1 $hsid1
+    echo "$output"
+    [ "$status" -eq 0 ]
+
+    local hsid2=$(host_set_id $NEW_HOST_SET2 $hcid)
+    run assoc_host_set_host $hid2 $hsid2
+    echo "$output"
+    [ "$status" -eq 0 ]
+}
+
+@test "boundary/host-set/add-host: verify all host sets contain created hosts" {
+    local hcid=$(host_catalog_id $NEW_HOST_CATALOG $DEFAULT_P_ID)
+    local hid1=$(host_id $NEW_HOST1 $hcid)
+    local hid2=$(host_id $NEW_HOST2 $hcid)
+
+    local hsid1=$(host_set_id $NEW_HOST_SET1 $hcid)
+    run host_set_has_host_id $hid1 $hsid1
+    echo "$output"
+    [ "$status" -eq 0 ]
+
+    local hsid2=$(host_set_id $NEW_HOST_SET2 $hcid)
+    run host_set_has_host_id $hid2 $hsid2
+    echo "$output"
+    [ "$status" -eq 0 ]
+}
+
+@test "boundary/target: can add created host set to created target" {
+    local hcid=$(host_catalog_id $NEW_HOST_CATALOG $DEFAULT_P_ID)
+    local hsid1=$(host_set_id $NEW_HOST_SET1 $hcid)
+    local tid=$(target_id_from_name $DEFAULT_P_ID $TGT_NAME)
+    run assoc_host_sources $tid $hsid1
+    echo "$output"
+    [ "$status" -eq 0 ]
+}
+
+@test "boundary/target: validate only $NEW_HOST1 host source present - JSON" {
```

Comment:
is `only` true here? Is that method just verifying that the specified host source is attached the target, not that it's the ONLY host source? (The same question applies to the other tests that say "only")

Reply:
Looking at some of the tests that mention `only`, that isn't always true. I'll fix the wording